commit 3aba8a7e10e509d4a321adb0b7540a73be023c10
Author: Fabian Vogt <fabian@ritter-vogt.de>
Date:   Sun Mar 17 12:17:54 2019 +0100

    Fix breeze dialog background with Qt 5.12.2
    
    Summary:
    style elements without type="text/css" were ignored before, but now they act as
    if type was set. Set id and style properly to restore the working behaviour.
    
    BUG: 405548
    
    Test Plan:
    Installed new files on a system with Qt 5.12.2, wiped the SVG cache
    and it looks fine again.
    
    Reviewers: #plasma, rooty
    
    Reviewed By: rooty
    
    Subscribers: acooligan, filipf, heikobecker, falqueto, rooty, Zren, arojas, rikmills, asturmlechner, kde-frameworks-devel, plasma-devel
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D19821

diff --git a/src/desktoptheme/breeze/dialogs/background.svgz b/src/desktoptheme/breeze/dialogs/background.svgz
index 32f82faee..78308d067 100644
Binary files a/src/desktoptheme/breeze/dialogs/background.svgz and b/src/desktoptheme/breeze/dialogs/background.svgz differ
diff --git a/src/desktoptheme/breeze/translucent/dialogs/background.svgz b/src/desktoptheme/breeze/translucent/dialogs/background.svgz
index 824ce3ca1..e74bed82f 100644
Binary files a/src/desktoptheme/breeze/translucent/dialogs/background.svgz and b/src/desktoptheme/breeze/translucent/dialogs/background.svgz differ
diff --git a/src/desktoptheme/breeze/translucent/widgets/tooltip.svgz b/src/desktoptheme/breeze/translucent/widgets/tooltip.svgz
index eb27d2e4f..fcdda89e3 100644
Binary files a/src/desktoptheme/breeze/translucent/widgets/tooltip.svgz and b/src/desktoptheme/breeze/translucent/widgets/tooltip.svgz differ
diff --git a/src/desktoptheme/breeze/widgets/tooltip.svgz b/src/desktoptheme/breeze/widgets/tooltip.svgz
index 70d3892a5..03d326963 100644
Binary files a/src/desktoptheme/breeze/widgets/tooltip.svgz and b/src/desktoptheme/breeze/widgets/tooltip.svgz differ
